app/logical/bulk_update_request_processor.rb

```diff
@@ -240,6 +266,8 @@ def to_dtext
         else
           "nuke {{#{args[0]}}}"
         end
+      when :deprecate, :undeprecate
+        "#{command.to_s.tr("_", " ")} [[#{args[0]}]]"
```

Comment:
The `.tr("_", " ")` shouldn't be necessary, since neither command has an underscore.